dev-docs/bidders/hypelab.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: HypeLab
+description: Prebid HypeLab Bidder Adapter
+pbjs: true
+pbs: false
+biddercode: hypelab
+sidebarType: 1
+media_types: banner
+userIds: all
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name | Scope | Description | Example | Type |
+|---------------|----------|------------------|---------|----------|
+| `property_slug` | required | The property slug | `prebid` | `string` |
+| `placement_slug` | required | The placement slug | `test_placement` | `string` |
```

Comment:
It's always nice to have an example configuration to see how it should look like

Reply:
Done!